openstack/compute/v2/flavors/requests.go

```diff
@@ -162,6 +162,36 @@ func ListAccesses(client *gophercloud.ServiceClient, id string) pagination.Pager
 	})
 }
 
+// AddAccessOptsBuilder allows extensions to add additional parameters to the
+// AddAccess requests.
+type AddAccessOptsBuilder interface {
+	ToAddAccessMap() (map[string]interface{}, error)
```

Comment:
@jtopjian I think these always need to be of the form `To<Resource><Action><Return Type>` (i.e. `ToFlavorAddAccessMap`)

Reply:
Yes, you're right. I've fixed this in #688. Sorry about that.